rclcpp/QUALITY_DECLARATION.md

```diff
@@ -121,6 +121,8 @@ This includes:
 
 Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
 
+Current coverage statistics can be viewed (79%) [here](https://ci.ros2.org/job/ci_linux_coverage/lastBuild/cobertura/src_ros2_rclcpp_rclcpp_src_rclcpp/). Current coverage statistics need to be improved to reach a higher quality level.
```

Comment:
Fix coverage links with lastSuccessfulBuild, also instead of listing the exact percentage which will change all the time. Say instead that coverage is being tracked, but it is not yet above 95%.

Fix here and below.